docs/hour-of-code-2021.html

```diff
@@ -126,182 +139,250 @@
-                            "
+                            id="carousel-link"
+                            target="_blank"
+                            rel="noreferrer"
+                            class="btn"
+                            href=""
+                            ><h5 id="carousel-text" class="display-font"></h5
                         ></a>
                     </div>
                 </div>
+                <script type="application/javascript">
+                    let slides = [
+                        {
+                            img: '/static/hour-of-code/2021/hoc-2020.png',
+                            link: 'https://arcade.makecode.com/hour-of-code',
+                            text: 'Hour of Code™ 2020'
+                        },
+                        {
+                            img: '/static/hour-of-code/2021/beginner-skillmap.png',
+                            link: 'https://arcade.makecode.com/--skillmap#beginner',
+                            text: 'Launch Skillmap'
+                        },
+                        { img: '', link: '', text: 'Launch Skillmap' },
+                        { img: '', link: '', text: 'Launch Skillmap' },
+                        { img: '', link: '', text: 'Hour of Code™ 2021' },
+                        {
+                            img: '/static/hour-of-code/2021/microbit.png',
+                            link: 'https://makecode.microbit.org',
+                            text: 'Go to micro:bit'
+                        }
+                    ];
+
+                    let globalInd = 0;
+                    let intervalSpeed = 4000;
+
+                    changeCarousel(0);
+
+                    function intervalFunction() {
+                        globalInd = (globalInd + 1) % 6;
+                        changeCarousel(globalInd);
+                    }
+
+                    let transition = setInterval(
+                        intervalFunction,
+                        intervalSpeed
+                    );
+
+                    function onListItemClicked(ind) {
+                        clearInterval(transition);
+                        globalInd = ind;
+                        changeCarousel(globalInd);
+                        transition = setInterval(
+                            intervalFunction,
+                            intervalSpeed
+                        );
+                    }
+
+                    function changeCarousel(ind) {
+                        let linkElems =
+                            document.getElementsByClassName('list-item');
+                        let carouselBody =
+                            document.getElementById('carousel-body');
+                        let carouselLink =
+                            document.getElementById('carousel-link');
+                        let carouselText =
+                            document.getElementById('carousel-text');
+
+                        [].forEach.call(linkElems, function (el) {
```

Comment:
`linkElems.forEach(el => {`? we don't usually use call syntax

Reply:
I tried to swap this out for your suggestion, but it broke the script and I'm not 100% sure why. Feel free to tinker with the script if you'd like!

Reply:
@jwunderl I asked Sonia to merge the change due to speed up. If you could help with the code review change that would be awesome. Thanks for the quick code review.